_posts/2020-11-12-quarkiverse.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: Quarkiverse - The Harbour of Community Extensions
+date: 2020-11-05
+tags: extensions quarkiverse
+synopsis: Entering a world of extensions
+author: gastaldi
+---
+
+Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, as well as an ever growing ecosystem of its extensions. At the beginning we were accepting all the contributed extensions in the core Quarkus repository. Eventually though it grew too large and lead to a maintenance overhead. In addition to that, in some cases it's simply not always sensible to include an extension into the core Quarkus repository.
+
+Some have since then created their own repositories external to Quarks, but others just never happened because not everyone wants the overhead of maintaining a build setup while in other cases we actually had an idea for an extension but not solidified enough yet to be included in core.
```

Comment:
```suggestion
Some have since then created their own repositories external to Quarkus, but others just never happened because not everyone wants the overhead of maintaining a build setup. Even the Quarkus core developers sometimes had ideas for an extension but not solid enough to be included in the Quarkus core repository.
```